src/error.rs

```diff
@@ -42,6 +42,7 @@ impl Error {
         Self::new(Kind::ActiveBlobExists)
     }
 
+    #[cfg(feature = "async-io-rio")]
```

Comment:
I think, this is not good. If this generates warning, then it is better to mark it with `#[allow(dead_code)]`

Reply:
Why is it no good? IMO this way we'll find out when it really becomes dead_code, so it's better to conditionally include it